packages/components/src/slot-fill2/README.md

```diff
@@ -0,0 +1,66 @@
+# Slot Fill
+
+Slot and Fill are a pair of components which enable developers to render elsewhere in a React element tree, a pattern often referred to as "portal" rendering. It is a pattern for component extensibility, where a single Slot may be occupied by an indeterminate number of Fills elsewhere in the application.
+
+Slot Fill is heavily inspired by the [`react-slot-fill` library](https://github.com/camwest/react-slot-fill), but uses [React's own portal rendering API](https://reactjs.org/docs/portals.html).
+
+## Usage
+
+At the root of your application, you must render a `SlotFillProvider` which coordinates Slot and Fill rendering.
+
+Then, render a Slot component anywhere in your application, giving it a name.
+
+Any Fill will automatically occupy this Slot space, even if rendered elsewhere in the application.
+
+You can either use the Fill component directly, or a wrapper component type as in the below example to abstract the slot name from consumer awareness.
+
+```jsx
+import { SlotFillProvider, Slot, Fill, Panel, PanelBody } from '@wordpress/components';
+
+const MySlotFillProvider = () => {
+	const MyPanelSlot = () => (
+		<Panel header="Panel with slot">
+			<PanelBody>
+				<Slot name="MyPanelSlot"/>
+			</PanelBody>
+		</Panel>
+	);
+	
+	MyPanelSlot.Content = () => (
+		<Fill name="MyPanelSlot">
+			Panel body
+		</Fill>
+	);
+
+	return (
+		<SlotFillProvider>
+			<MyPanelSlot />
+			<MyPanelSlot.Content />
+		</SlotFillProvider>
+	);
+};
+```
+
+There is also `createSlotFill` helper method which was created to simplify the process of matching the corresponding `Slot` and `Fill` components:
+
+```jsx
+const { Fill, Slot } = createSlotFill( 'Toolbar' );
+
+const ToolbarItem = () => (
+	<Fill>
+		My item
+	</Fill>
+);
+
+const Toolbar = () => (
+	<div className="toolbar">
+		<Slot />
+	</div>
+); 
+```
+
+## Props
+
+The `SlotFillProvider` component does not accept any props.
+
+Both `Slot` and `Fill` accept a `name` string prop, where a `Slot` with a given `name` will render the `children` of any associated `Fill`s.
```

Comment:
I removed this README.md file as the contents of `slot-fill2` were moved over to `slot-fill`. But, the reason to remove that was because `<Slot bubblesVirtually />` didn't accept children prop. Since this refactor only considers that use case, it didn't make sense keeping that in the readme.

But now it's possible to access fills through `useSlot`. See https://github.com/WordPress/gutenberg/pull/19242#discussion_r381127219